package.json

```diff
@@ -36,8 +36,8 @@
     "browserify": "node_modules/browserify/bin/cmd.js test/*.js --outfile lib/test/bundle.js",
     "bookmarklet": "scripts/bookmarklet",
     "clean": "rimraf dist",
-    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js",
-    "jshint": "jshint scripts/bookmarklet scripts/build **/*.js",
+    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js **/**/*.js",
```

Comment:
Too late to the party once again.  I would rather just:

    "jscs": "jscs scripts/bookmarklet scripts/build .",

which will allow it to work in Windoze too.